src/src/PluginStructs/P129_data_struct.cpp

```diff
@@ -36,7 +36,8 @@ bool P129_data_struct::plugin_init(struct EventStruct *event) {
   return false;
 }
 
-const uint32_t P129_data_struct::getChannelState(uint8_t offset, uint8_t size) const {
+const uint32_t P129_data_struct::getChannelState(uint8_t offset,
```

Comment:
Why return a const int?
It isn't a reference, so it is a deepcopy anyway.

Reply:
Corrected